ietf-scitt-charter.md

```diff
@@ -68,11 +76,12 @@ The WG shall specify a standard format for authenticity data returned from the t
 
 ## Generic Protocol Bindings for Information Model and Interaction Models
 
-The WG shall standardize request-response interactions ("external API") and potentially other generic interaction schemes provided to various actors to interact with the supply chain ecosystem. This includes standardizing inter-component messages (based on the interaction models) between supply chain actors to support easy reference implementations of SCITT building blocks by various organizations and easy industry-wide adaptation.
+The WG shall standardize request-response interactions ("external API") and potentially other generic interaction schemes provided to various actors to interact with the supply chain ecosystem. This includes standardizing inter-component messages (based on the interaction models) and payload serialization between supply chain actors to support easy reference implementations of SCITT building blocks by various organizations and easy industry-wide adaptation.
```

Comment:
```suggestion
The WG shall standardize request-response interactions ("external API") and potentially other generic interaction schemes provided to various  external entities to interact with the supply chain ecosystem. This includes standardizing inter-component messages (based on the interaction models) and payload serialization between supply chain actors to support easy reference implementations of SCITT building blocks by various organizations and easy industry-wide adaptation.
```

Reply:
Do we want to change from actors now? In the meeting yesterday, actors seemed to have reached a "good enough" consensus, including chairs.

Reply:
Either change it to external actors to distinguish it clearly from SCITT actors or call it external entities for external parties, either one is fine with me

Reply:
What I am reading is that the use of actor in "Architectural Model: Actors, Interactions, Terminology" has different semantics than the use of actor here in "Generic Protocol Bindings for Information Model and Interaction Models".

That is why @yogeshbdeshpande proposes "external entities" instead of "actors".

My question: Are the semantics actually different? Either way it might make sense. If we mean "issuer", "notary", "consumer", etc. here, we are fine.